CHANGES.rst

```diff
@@ -16,15 +16,9 @@ datamodels
 
 - Force data model type setting on save [#4318]
 
-- Deprecate ``MIRIRampModel`` [#4328]
+- Deprecate MIRIRampModel [#4328]
 
-- Make ``memmap=False`` be the default in ``datamodels`` [#4445]
-
-- Update schemas to add the ``id`` field and switch relative references
```

Comment:
Are these lines getting dropped because of a conflict/mismatch in versions of the file?

Reply:
I often have trouble with the change log.  git tells me there's no conflict, but lines mysteriously disappear.  I'll try again with this file.